### PR TITLE
Fixed text file error and added file name

### LINKED ISSUE #777: [Add Text] Error when selected text file
Snackbar shows message `Error Occurred `when text file is selected before PDF file

### Steps to reproduce
* Add Text
* Select Text File (before selecting PDF file)

### Expected behaviour
File should be selected and path to be shown on `select` button.

### Actual behaviour
Snackbar shows message `Error Occurred `

### PR DESCRIPTION
**Pull request by shldhll**
# Description

Fixed the "Error Occurred" when trying to convert  ```Text to PDF ``` before converting  ```Images to PDF ```.

```Select Text File``` button now shows file name after user selects the file to be converted.
* The bottom TextView which displayed the file name has been removed.

screenshot1

screenshot2

Fixes #777 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/util/TextToPdfAsync.java [hunk 1]
```diff
@@ -39,9 +45,16 @@ protected void onPreExecute() {
         mOnPDFCreatedInterface.onPDFCreationStarted();
     }
 
+    private void createParentDir() {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/TextToPdfAsync.java]: don't add utility fiunctions here. these should be in "FileUtils" or "DirectoryUtils"

**Comment by shldhll** [comment]
> just on minor change that I have mentioned.
> Other than that, looks good!

File: app/src/main/java/swati4star/createpdf/util/TextToPdfAsync.java (hunk 1 above)
Comment (shldhll) [reply app/src/main/java/swati4star/createpdf/util/TextToPdfAsync.java]: Removed createParentDir() function and started using DirectoryUtils function instead.

File: app/src/main/java/swati4star/createpdf/util/TextToPdfAsync.java [hunk 2]
```diff
@@ -56,4 +56,4 @@ protected void onPostExecute(Object o) {
         mOnPDFCreatedInterface.onPDFCreated(mSuccess);
     }
 
-}
+}
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/TextToPdfAsync.java]: remove this file from commit.
there is no change here

Comment (shldhll) [reply app/src/main/java/swati4star/createpdf/util/TextToPdfAsync.java]: I have deleted the file. Is this the correct way to remove it from the commit?

**Pull request closed without merging**